src/test/lib/operators/get_table_test.cpp

```diff
@@ -244,12 +256,8 @@ TEST_F(OperatorsGetTableTest, PrunedChunksCombined) {
   EXPECT_FALSE(delete_all->execute_failed());
   context->commit();
 
-  /*
-   * Not setting cleanup commit ids is intentional,
-   * because we delete the chunks manually for this test.
-  */
-
-  // Delete chunks physically
+  // Not setting cleanup commit ids is intentional, because we delete the chunks manually for this test.
+  // Delete chunks physically.
```

Comment:
The sentence with physically directly following the sentence with manually is confusing.